From 864e17068ce9acf418d42a625141884293170952 Mon Sep 17 00:00:00 2001
From: Lennart Poettering <lennart@poettering.net>
Date: Fri, 10 Oct 2014 11:11:25 +0200
Subject: [PATCH] nspawn: actually allow access to /dev/net/tun in the
 container

It's not sufficient to just copy the device node over, we need to update
the policy for it too.
---
 src/nspawn/nspawn.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/nspawn/nspawn.c b/src/nspawn/nspawn.c
index da4c116f36..f04d326131 100644
--- a/src/nspawn/nspawn.c
+++ b/src/nspawn/nspawn.c
@@ -1558,6 +1558,7 @@ static int register_machine(pid_t pid, int local_ifindex) {
                                           "/dev/random", "rwm",
                                           "/dev/urandom", "rwm",
                                           "/dev/tty", "rwm",
+                                          "/dev/net/tun", "rwm",
                                           /* Allow the container
                                            * access to ptys. However,
                                            * do not permit the
